python/lsst/meas/extensions/photometryKron/photometryKron.cc

```diff
@@ -0,0 +1,140 @@
+    cls.attr("NO_MINIMUM_RADIUS") = py::cast(static_cast<int>(KronFluxAlgorithm::NO_MINIMUM_RADIUS));
+    cls.attr("NO_FALLBACK_RADIUS") = py::cast(static_cast<int>(KronFluxAlgorithm::NO_FALLBACK_RADIUS));
+    cls.attr("BAD_RADIUS") = py::cast(static_cast<int>(KronFluxAlgorithm::BAD_RADIUS));
+    cls.attr("USED_MINIMUM_RADIUS") = py::cast(static_cast<int>(KronFluxAlgorithm::USED_MINIMUM_RADIUS));
+    cls.attr("USED_PSF_RADIUS") = py::cast(static_cast<int>(KronFluxAlgorithm::USED_PSF_RADIUS));
+    cls.attr("SMALL_RADIUS") = py::cast(static_cast<int>(KronFluxAlgorithm::SMALL_RADIUS));
+    cls.attr("BAD_SHAPE") = py::cast(static_cast<int>(KronFluxAlgorithm::BAD_SHAPE));
+    cls.attr("N_FLAGS") = py::cast(static_cast<int>(KronFluxAlgorithm::N_FLAGS));
+
+    cls.def(py::init<KronFluxAlgorithm::Control const &, std::string const &, afw::table::Schema &,
+                     daf::base::PropertySet &>(),
+            "ctrl"_a, "name"_a, "schema"_a, "metadata"_a);
+
+    cls.def("measure", &KronFluxAlgorithm::measure, "measRecord"_a, "exposure"_a);
+    cls.def("measureForced", &KronFluxAlgorithm::measureForced, "measRecord"_a, "exposure"_a, "refRecord"_a,
+            "refWcs"_a);
+    cls.def("fail", &KronFluxAlgorithm::fail, "measRecord"_a, "error"_a = NULL);
+}
+
+using PyKronAperture = py::class_<KronAperture>;
+
+/**
+ * Wrap templated methods of KronAperture
+ *
+ * @tparam ImageT  Image-like class, such as lsst::afw::image::MaskedImage<float>
+ * @param cls  pybind11 class wrapping KronAperture
+ */
+template <typename ImageT>
+void declareKronApertureTemplatedMethods(PyKronAperture &cls) {
+    cls.def_static("determineRadius", &KronAperture::determineRadius<ImageT>, "image"_a, "axes"_a, "center"_a,
+            "ctrl"_a);
+    cls.def("measureFlux", &KronAperture::measureFlux<ImageT>, "image"_a, "nRadiusForFlux"_a,
+            "maxSincRadius"_a);
+}
+
+void declareKronAperture(py::module &mod) {
+    PyKronAperture cls(mod, "KronAperture");
+
+    cls.def(py::init<afw::geom::Point2D const &, afw::geom::ellipses::BaseCore const &, float>(), "center"_a,
+            "core"_a, "radiusForRadius"_a = std::nanf(""));
+    cls.def(py::init<afw::table::SourceRecord const &, float>(), "source"_a,
+            "radiusForRadius"_a = std::nanf(""));
+    cls.def(py::init<afw::table::SourceRecord const &, afw::geom::AffineTransform const &, double, float>(),
+            "reference"_a, "refToMeas"_a, "radius"_a, "radiusForRadius"_a = std::nanf(""));
+
+    cls.def_static("getKronAxes", &KronAperture::getKronAxes, "shape"_a, "transformation"_a, "radius"_a);
+
+    cls.def("getX", &KronAperture::getX);
+    cls.def("getY", &KronAperture::getY);
+    cls.def("getRadiusForRadius", &KronAperture::getRadiusForRadius);
+    cls.def("getCenter", &KronAperture::getCenter);
+    cls.def("getAxes", (afw::geom::ellipses::Axes & (KronAperture::*)()) & KronAperture::getAxes,
+            py::return_value_policy::copy);
```

Comment:
For small objects that are returned by `& const` in C++, my preference is to copy, because the Python philosophy that users who mess with things they shouldn't deserve what they get doesn't really extend to segfaults caused by invalidating the invariants of a C++ class.  But it looks like the C++ call here returns by non-const reference (is that right?), for which should be fine to use `reference_internal`.

A side note: while it's fine to put spaces around `&` when declaring a reference, please don't put a space after it when taking the address of something.